### PR TITLE
Setup for generating our own RihannaUI image

### PR DESCRIPTION
**Pull request by mackshkatz**
- [x] upgrade phoenix
- [x] work some magic around postgrex configuration (thank you greatly @rubysolo )
- [x] set up circle for this project
- [x] set up deploy script for this project
- [x] I have already provisioned the repo in ECR, so I just need to push up the image generated from the Dockerfile once this is officially all good to go
- [x] Update the k8s service to point to the new image in ECR

### CODE REVIEW

File: config/prod.exs [hunk 1]
```diff
@@ -21,6 +21,11 @@ config :rihanna_ui, RihannaUIWeb.Endpoint,
 # Do not print debug messages in production
 config :logger, level: :info
 
+# Configure your database
+config :rihanna_ui, RihannaUI.Repo,
+  adapter: Ecto.Adapters.Postgres,
+  url: System.get_env("DATABASE_URL", "postgres://localhost/violet_dev")
```
Comment (chakeresa) [inline config/prod.exs:27]: Prod defaults to localhost violet_dev?

File: lib/rihanna_ui_web/router.ex [hunk 2]
```diff
@@ -16,6 +16,13 @@ defmodule RihannaUIWeb.Router do
   scope "/", RihannaUIWeb do
     pipe_through :browser # Use the default browser stack
 
+    # TODO
```
Comment (chakeresa) [inline lib/rihanna_ui_web/router.ex:19]:
```suggestion
    # TODO: https://github.com/BlinkerGit/rihanna_ui/issues/2
```

File: mix.exs [hunk 3]
```diff
@@ -5,10 +5,10 @@ defmodule RihannaUI.Mixfile do
     [
       app: :rihanna_ui,
       version: "0.5.2",
-      elixir: "~> 1.4",
```
Comment (chakeresa) [inline mix.exs:8]: 👵🏻

File: mix.exs [hunk 4]
```diff
@@ -26,22 +26,26 @@ defmodule RihannaUI.Mixfile do
...
+      {:ecto_sql, "~> 3.1"},
+      {:postgrex, ">= 0.15.6"},
+      {:phoenix_html, "~> 2.13"},
+      {:jason, "~> 1.2.2"},
+      {:phoenix_live_reload, "~> 1.2.4", only: :dev},
+      {:gettext, "~> 0.18.2"},
+      {:plug_cowboy, "~> 2.0"},
+      {:plug, "~> 1.7"},
+      {:rihanna, ">= 2.2.0"},
+      {:credo, "~> 1.4", only: [:dev, :test], runtime: false}
```
Comment (chakeresa) [inline mix.exs:48]: Consider alphabetizing so it's easy to check for stuff / avoid merge conflicts (ya know, on all that work we're constantly doing to this repo).

File: config/prod.exs (hunk 1 above)
Comment (rubysolo) [reply config/prod.exs:27]: yeah, prod should probably die if this var is missing, like in violet.

**Review by chakeresa — APPROVED** [review]
Nice! Left 1 or 2 comments -- take a look at those, but otherwise looks good to me. Nicely done.

File: config/prod.exs (hunk 1 above)
Comment (mackshkatz) [reply config/prod.exs:27]: good eye, this was a relic of debugging locally while messing with env vars

File: mix.exs (hunk 3 above)
Comment (mackshkatz) [reply mix.exs:8]: she did her job, she can sleep easy now

**Pull request merged**